Code/ThirdParty/sal/sal.h

```diff
@@ -0,0 +1,2961 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/***
+*sal.h - markers for documenting the semantics of APIs
```

Comment:
Are you trying to trigger my PTSD ?

Reply:
Not intentionally. But now that you mention it, this is a nice added bonus ;-)